### PR TITLE
Fix Multi Net attachmetns with CNI post

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
The NNCP version has to be v1alpha1 since latest knmstate didn't make it
to CNAO -> HCO.
Is no longer needed to reset k8s context namespace after HCO deploy.
Also looks like production is missing `doc` command but we cannot fis it at this PR.

**Does this PR fix any issue?** _(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)_:

> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown [hunk 1]
```diff
@@ -156,6 +156,7 @@ To start it up with two nodes and one secondary NIC and install NetworkManager >
 ```bash
 git clone https://github.com/kubevirt/kubevirtci
 cd kubevirtci
+git reset d5d8e3e376b4c3b45824fbfe320b4c5175b37171 --hard
```
Comment (maiqueb) [inline _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown:159]: A comment justifying this line might be a good thing. Or not (git blame will give us the required info).

Just a thought.

File: _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown [hunk 2]
```diff
@@ -171,7 +172,6 @@ needed to have a functional KubeVirt with all the features including the ones we
 ```bash
 curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/deploy/deploy.sh | bash
 kubectl wait hco -n kubevirt-hyperconverged kubevirt-hyperconverged --for condition=Available --timeout=500s
-kubectl config set-context --current --namespace=default # Put back the previous context namespace
```
Comment (maiqueb) [inline _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown:174]: I would prefer to understand why this is no longer needed; on the commit msg you explain this is not needed, but omit the reason. Could you add it to the commit msg ?

Comment (qinqon) [reply _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown:174]: Bug at HCO https://github.com/kubevirt/hyperconverged-cluster-operator/pull/890

Comment (maiqueb) [reply _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown:174]: I'd still rather have a TL;DR; explanation of it than a link to a bug.

Comment (qinqon) [reply _posts/2020-10-21-Multiple-Network-Attachments-with-bridge-CNI.markdown:174]: Added the PR to the commit msg

**Review by maiqueb — APPROVED** [review]
Decent compromise.

**Pull request merged**